google/cloud/internal/parse_rfc3339.cc

```diff
@@ -94,7 +94,7 @@ std::chrono::system_clock::time_point ParseDateTime(
   if (hours < 0 || hours >= kHoursInDay) {
     ReportError(timestamp, "Out of range hour.");
   }
-  if (minutes < 0 || minutes >= kMinutesInHour - 1) {
+  if (minutes < 0 || minutes >= kMinutesInHour) {
```

Comment:
Sorry I missed that.

Reply:
I think I am more sorry for introducing the problem.